podcasts/Onboarding/Plus/SubscriptionPriceAndOfferView.swift

```diff
@@ -17,64 +17,58 @@ struct SubscriptionPriceAndOfferView: View {
 
     var body: some View {
         VStack(alignment: .leading, spacing: 10) {
-            SubscriptionBadge(tier: product.identifier.subscriptionTier)
-            if let offerDescription = offerDescription(for: product) {
-                Text(offerDescription)
-                    .lineLimit(1)
-                    .fixedSize(horizontal: true, vertical: false)
```

Comment:
The fixed size will ensure the view always maintains it content width even if the container is too small. If this gets larger than its container though, SwiftUI will expand the containers width to fit it.